src/pages/sighting/encounters/AddAnnotationsDialog.jsx

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from 'react';
+import { get, omit } from 'lodash-es';
+
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogActions from '@material-ui/core/DialogActions';
+import Grid from '@material-ui/core/Grid';
+
+import useAddAnnotationsToSightingEncounter from '../../../models/encounter/useAddAnnotationsToSightingEncounter';
+import useAddAnnotationsToAGSEncounter from '../../../models/assetGroupSighting/useAddAnnotationsToAGSEncounter';
+import StandardDialog from '../../../components/StandardDialog';
+import Button from '../../../components/Button';
+import Alert from '../../../components/Alert';
+import AnnotatedPhotograph from '../../../components/AnnotatedPhotograph';
+
+export default function AddAnnotationsDialog({
+  sightingData,
+  pending,
+  encounterId,
+  onClose,
+}) {
+  const [selectedAnnotations, setSelectedAnnotations] = useState([]);
+
+  const {
+    addAnnotationsToSightingEncounter,
+    error: addToSightingEncounterError,
+    isLoading: addToSightingEncounterLoading,
+  } = useAddAnnotationsToSightingEncounter();
+
+  const {
+    addAnnotationsToAGSEncounter,
+    error: addToAGSEncounterError,
+    isLoading: addToAGSEncounterLoading,
+  } = useAddAnnotationsToAGSEncounter();
+
+  const error = pending
+    ? addToAGSEncounterError
+    : addToSightingEncounterError;
+  const loading = pending
+    ? addToAGSEncounterLoading
+    : addToSightingEncounterLoading;
+
+  const open = Boolean(encounterId);
+  const onCloseDialog = () => {
+    setSelectedAnnotations([]);
+    onClose();
+  };
+
+  const sightingId = get(sightingData, 'guid');
+  const assets = get(sightingData, 'assets', []);
+  const annotations = assets.reduce((acc, asset) => {
+    const assetAnnotations = get(asset, 'annotations', []);
+    const amendedAssetAnnotations = assetAnnotations.map(a => ({
+      ...a,
+      ...omit(asset, ['annotations', 'guid']),
```

Comment:
why are we getting ride of guids?

Reply:
because the asset guid was overwriting the annotation guid, and we only need the latter below